docs/tools/vdb_table/data/aperturedb.json

```
@@ -127,33 +127,33 @@
     "comment": ""
   },
   "ephemeral": {
-    "support": "",
+    "support": "partial",
     "source_url": "",
-    "comment": ""
+    "comment": "Not documented externally."
   },
   "sharding": {
-    "support": "",
+    "support": "partial",
     "source_url": "",
-    "comment": ""
+    "comment": "Not documented externally."
   },
   "doc_size": {
     "bytes": 0,
-    "unlimited": false,
-    "source_url": "",
-    "comment": ""
+    "unlimited": true,
+    "source_url": "https://docs.aperturedata.io/administration/Configuring%20ApertureDB",
+    "comment": "You can store large or small files but need to change message size for adding or querying files larger than 256MB. For very large files like GBs of video, it will require optimizations to send over network or would be slow."
   },
   "vector_dims": {
     "value": 0,
     "unlimited": true,
-    "source_url": "",
-    "comment": ""
+    "source_url": "https://docs.aperturedata.io/query_language/reference/descriptor_commands/desc_set_commands/AddDescriptorSet",
+    "comment": "Unlimited. Currently tested up to 4096 dims"
   },
   "index_types": {
     "value": [
-      "HNSW"
+      "HNSW", "FLAT", "IVF"
     ],
-    "source_url": "",
-    "comment": ""
+    "source_url": "https://docs.aperturedata.io/query_language/reference/descriptor_commands/desc_set_commands/AddDescriptorSet",
```

Hi @vishakha041, I recommend specifying this exact subsection in the URL so that the reader directly sees the part that mentions "the supported engines are HNSQ, IVF and Flat". 
```suggestion
    "source_url": "https://docs.aperturedata.io/query_language/Reference/descriptor_commands/desc_set_commands/AddDescriptorSet#details",
```